src/EditorFeatures/Core/Implementation/BraceMatching/AbstractEmbeddedLanguageBraceMatcher.cs

```diff
@@ -15,17 +15,20 @@ internal abstract class AbstractEmbeddedLanguageBraceMatcher : IBraceMatcher
         public async Task<BraceMatchingResult?> FindBracesAsync(
             Document document, int position, CancellationToken cancellationToken)
         {
-            var languageProvider = document.GetLanguageService<IEmbeddedLanguageProvider>();
-            foreach (var language in languageProvider.GetEmbeddedLanguages())
+            var languagesProvider = document.GetLanguageService<IEmbeddedLanguagesProvider>();
+            if (languagesProvider != null)
```

Comment:
note: this is just paranoia.  AbstractEmbeddedLanguageBraceMatcher should only be active for C#/VB.  But given our history of doing things like this and blowing up for TS/F#, i'd prefer to be paranoid.